### DISCUSSION TITLE
dcm4che/postgres-dcm4chee same Tag, but different Image Id?

### DISCUSSION THREAD

**Discussion by dirkooms** [dcm4che/dcm4chee-arc-light#3422] [Q&A] [answered]
Last April I installed an arc-light on a synology. All worked fine.

Now I'm doing the same thing (same docker-compose.yml), but it doesn't work. I noticed that in the db log there were errors that the tables/relations queue_msg and export_task were not present (the old install had 38 tables, the new one only 34 tables).
Then I looked to the images of postgres-dcm4chee. They both have the same Tag '13.1-23', but different Image Id's. The old (good) Image Id is 56c66c402a9e, the new (bad) Image Id is 53b0dd94358d.

I'm a docker newbie, but is it normal that images with the same Tag can have different Image Id's. If not, what is happening  here and how can it be resolved?
Any tip will be highly appreciated!

![Screenshot from 2021-10-14 15-39-15](https://user-images.githubusercontent.com/5609307/137331798-4d8810bb-d1f1-43cf-8b6f-fdcf85cc4067.png)

![Screenshot from 2021-10-14 15-39-42](https://user-images.githubusercontent.com/5609307/137331828-4d4bc5bb-b4a3-4f03-b0eb-f6f95170cb2c.png)
.

**Comment by gunterze** [comment, marked as answer]
Seems, I accidentally overwrote `dcm4che/postgres-dcm4chee:13.1-23` with a version for 5.24.x.
`dcm4che/postgres-dcm4chee:13.1-23` should now be restored.

**Comment by dirkooms** [comment]
Great, seems to be solved. At least I can see the missing tables again. FYI: the Image Id is a new one: 0405b69c7381 (different from both old and new).

Thanks for your quick response. Saved my day (and night).

**Reply by gunterze** [reply]
Didn't had a backup of old dcm4che/postgres-dcm4chee:13.1-23, so I built a new...
